src/Editable.php

```diff
@@ -109,21 +109,16 @@ public function init()
     public function run()
     {
         $value = $this->value;
-        if ($this->hasModel()) {
-            $model = $this->model;
-            if ($value !== null) {
-                if (is_string($value)) {
-                    $show = ArrayHelper::getValue($model, $value);
-                } else {
-                    $show = call_user_func($value, $model);
-                }
-            } else {
-                $show = ArrayHelper::getValue($model, $this->attribute);
-            }
+        if ($this->hasModel() && $value === null) {
```

Comment:
Check whether `is_callable($value)` if returns true then it will execute this method which should return the value you want to display otherwise outputs the value of `$value`